test/dlc_tests/conftest.py

```diff
@@ -1029,6 +1031,45 @@ def _validate_pytorch_framework_version(request, image_uri, test_name, skip_dict
     return False
 
 
+@pytest.fixture(scope="session")
+def telemetry():
+    telemetry_test = os.getenv("TEST_TYPE", "UNDEFINED")
+    if is_mainline_context() and telemetry_test != "telemetry":
```

Comment:
Is the reason you're doing this so that telemetry test will run alongside EC2 in PR and in its dedicated job in pipeline context? I would leave a comment here explaining those assumptions

Reply:
Yes. I'll add a docstring explaining that.